field-sustainability-helper/src/index.css

```diff
@@ -1,3 +1,12 @@
 body {
   margin: 0;
 }
+
+.progressBar {
+  height: 20px;
```

Comment:
For future reference, I would recommend not hardcoding pixel values for width, height, etc. Using percentages or view height/view width would be better, but I know there are pixel values hardcoded elsewhere in the code so I don't think it's a huge deal